bindings/sundials4py/test/test_idas.py

```diff
@@ -0,0 +1,225 @@
+#!/bin/python
+# -----------------------------------------------------------------
+# Programmer(s): Cody J. Balos @ LLNL
+# -----------------------------------------------------------------
+# SUNDIALS Copyright Start
+# Copyright (c) 2025-2025, Lawrence Livermore National Security,
+# University of Maryland Baltimore County, and the SUNDIALS contributors.
+# Copyright (c) 2013-2025, Lawrence Livermore National Security
+# and Southern Methodist University.
+# Copyright (c) 2002-2013, Lawrence Livermore National Security.
+# All rights reserved.
+#
+# See the top-level LICENSE and NOTICE files for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+# SUNDIALS Copyright End
+# -----------------------------------------------------------------
+
+import pytest
+import numpy as np
+from sundials4py.core import *
+from sundials4py.idas import *
+from problems import AnalyticDAE
+from fixtures import *
+
+
+@pytest.mark.skipif(
+    sunrealtype == np.float32, reason="Test not supported for sunrealtype=np.float32"
+)
+def test_idas_ivp(sunctx):
+    ode_problem = AnalyticDAE()
+
+    ida = IDACreate(sunctx)
+    yy = N_VNew_Serial(2, sunctx)
+    yp = N_VNew_Serial(2, sunctx)
+
+    # y and y' initial conditions
+    ode_problem.set_init_cond(yy, yp, ode_problem.T0)
+
+    ls = SUNLinSol_SPGMR(yy, SUN_PREC_LEFT, 0, sunctx)
+
+    def resfn(t, yy, yp, rr, _):
+        return ode_problem.res(t, yy, yp, rr)
+
+    def psolve(t, yy, yp, rr, r, z, cj, delta, _):
+        return ode_problem.psolve(t, yy, yp, rr, r, z, cj, delta)
+
+    status = IDAInit(ida.get(), resfn, 0.0, yy, yp)
+    assert status == IDA_SUCCESS
+
+    status = IDASStolerances(ida.get(), 1e-4, 1e-4)
```

Comment:
Use `SUNREALTYPE_RTOL` and `SUNREALTYPE_ATOL`?
```suggestion
    status = IDASStolerances(ida.get(), SUNREALTYPE_RTOL, SUNREALTYPE_ATOL)
```